src/Microsoft.AspNet.OData.Shared/Builder/StructuralTypeConfiguration.cs

```diff
@@ -143,6 +144,22 @@ public PropertyInfo DynamicPropertyDictionary
             get { return _dynamicPropertyDictionary; }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether this type has instance annotations or not.
+        /// </summary>
+        public bool IsWithInstanceAnnotations
```

Comment:
>public [](start = 8, length = 6)

Should this be public? Do we even need it?  IsOpen is a characteristic of the EdmType, whether something has instance annotations is a capability of the CLR type.

Reply:
#Resolved

Reply:
This is not really needed, but I'm okay keeping it as a (minor) convenience function.

---
In reply to: [458402613](https://github.com/OData/WebApi/pull/2219#discussion_r458402613) [](ancestors = 458402613)